packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/user/UserEvent.java

```diff
@@ -0,0 +1,22 @@
+package software.uncharted.terarium.hmiserver.models.user;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.quarkus.runtime.annotations.RegisterForReflection;
+import lombok.Data;
+import software.uncharted.terarium.hmiserver.models.EventType;
+
+import java.util.UUID;
+
+@RegisterForReflection
+@Data
+public class UserEvent<S> {
```

Comment:
What is `S` meant to do? It doesn't seem to be used or registered anywhere.

Reply:
@chris-dickson why is this here?

Reply:
Curious why you made a new class for this.  Why not use `Event` that is nearly a clone to this.